pkg/agent/openflow/pipeline.go

```diff
@@ -599,7 +658,7 @@ func (f *featurePodConnectivity) conntrackFlows() []binding.Flow {
 			ConntrackTable.ofTable.BuildFlow(priorityNormal).
 				Cookie(cookieID).
 				MatchProtocol(ipProtocol).
-				Action().CT(false, ConntrackTable.ofTable.GetNext(), f.ctZones[ipProtocol]).
+				Action().CT(false, ConntrackTable.ofTable.GetNext(), f.ctZones[ipProtocol], getZoneSrcFieldName(f.connectUplinkToBridge), getZoneSrcRange(f.connectUplinkToBridge)).
```

Comment:
Define two global variables for FieldName and Range to avoid two calls everywhere?

Reply:
I checked the global variables in pkg/agent/openflow, all the configuration related variables are struct members, and we won't change the global variables in this package.
Thus I think we should keep this inside the struct to reduce the maintenance cost.
Do you think we should cache the result of the 2 functions to reduce below calls?
```
CtZoneField.GetNXFieldName()
CtZoneField.GetRange().ToNXRange()
```

Reply:
Yeah, you can also add two variables for featurePodConnectivity, but then you need to do that for multiple features.

Reply:
Added these 2 variables to connectivity/service/NP features.